### ISSUE TITLE
pdb not updated

### ISSUE THREAD

**Issue by vip32** [GitTools/GitLink#118]
as part of my CI build i run the gitlink. However i see the srcsrv gets created with the proper bitbucket things in it. however the pdb itself does not seem updated (however date was modified). how do i verify that the pdb has the right information?

with cvdump is still see all the local filenames in the pdb and no git info. what should i see in the pdb that gets updated. i suppose i don't have to distribute srcsrv files in my nuget packages.

any insight would be helpful

**Comment by GeertvanHorrik** [comment]
The pdb files remap the files from lopcal => remote. This is mostly visible when you open the pdb file inside a text editor (e.g. Notepad++) and scroll all the way down:

```
VERSION=2
SRCSRV: variables ------------------------------------------
SRCSRVVERCTRL=https
SRCSRVTRG=https://raw.github.com/GitTools/GitTools.Core/5cf4e7bd127b07a31f421b763afd234a07c1adba/%var2%
SRCSRV: source files ---------------------------------------
C:\CI_WS\Ws\78322\Source\GitTools_Core\src\SolutionAssemblyInfo.cs*src/SolutionAssemblyInfo.cs
C:\CI_WS\Ws\78322\Source\GitTools_Core\src\GitTools.Core\App_Packages\LibLog.4.2\LibLog.cs*src/GitTools.Core/App_Packages/LibLog.4.2/LibLog.cs
C:\CI_WS\Ws\78322\Source\GitTools_Core\src\GitTools.Core\Constants.cs*src/GitTools.Core/Constants.cs
C:\CI_WS\Ws\78322\Source\GitTools_Core\src\GitTools.Core\Testing\TestValues.cs*src/GitTools.Core/Testing/TestValues.cs
C:\CI_WS\Ws\78322\Source\GitTools_Core\src\GitTools.Core\Context\ContextBase.cs*src/GitTools.Core/Context/ContextBase.cs
```

As you can see, it shows that local paths should be mapped to the url part after the *. So this line:

```
C:\CI_WS\Ws\78322\Source\GitTools_Core\src\SolutionAssemblyInfo.cs*src/SolutionAssemblyInfo.cs
```

Will map `C:\CI_WS\Ws\78322\Source\GitTools_Core\src\SolutionAssemblyInfo.cs` to `https://raw.github.com/GitTools/GitTools.Core/5cf4e7bd127b07a31f421b763afd234a07c1adba/src/SolutionAssemblyInfo.cs`

**Comment by vip32** [comment]
hoi geert,
i see this mapping information with the RAWURL in the srcsrv file, but it does not appear in the updated pdb. should that be the case?

**Comment by GeertvanHorrik** [comment]
It should be in the updated pdb as well. Is there a public repro somewhere? Make sure that you are looking to the right pdb files.

**Comment by vip32** [comment]
no it's not public ;(
i see this behavior running local and in jenkins. however we use the cake aliases for gitlink, which justs calls the tool. not suspecting any issues with that. let me try the tool directly.

am using the latest stable, should a -pre work better in this case?

**Comment by vip32** [comment]
it should update the pdb next to where the srcsrv is created, right?

**Comment by GeertvanHorrik** [comment]
Yes, correct, it should update that specific pdb. What is the output of GitLink?

**Comment by xMilotas** [comment]
Hey,
any update on this issue? 
We still have the same problem - GitLink is not showing any error messages, but the pdb's are not updated.

**Comment by GeertvanHorrik** [comment]
Nope. Do you have an output for us to analyze?

**Comment by xMilotas** [comment]
Since the repository is private I had to cut out some stuff. 
This is what we get, when running GitLink with Cake. - We get the same output when starting the GitLink.exe directly. 
[GitLinkLog.txt](https://github.com/GitTools/GitLink/files/726872/GitLinkLog.txt)

**Comment by vip32** [comment]
Hoi Geert,

is there a way to enable more logging, set some kind of verbosity level? would be nice if git-link outputs the actual changes it made.

we really like to get this working, as we can save the hassle of setting up a symbol server.

**Comment by GeertvanHorrik** [comment]
I tried setting -debug yesterday, but it causes a bug on the parameter count checks. You can try setting -debug 1 to get around this bug. Can you let me know if that works?

**Comment by GeertvanHorrik** [comment]
@xMilotas Are you sure that the pdb's are not updated? Note that GitLink defaults to "Release | AnyCPU" configurations and it updates the pdb files on the original location.

**Comment by xMilotas** [comment]
When I use -debug it shows me the message

>   Verifying pdb file
  Created source server link file, updating pdb file '[path].pdb'

When I look at this file(which should be the correct one) with Notepad++, I still only see the old paths.
Is there anything else that I could look for when using -debug, that could give me any kind of information?
Thank you in advance.

**Comment by GeertvanHorrik** [comment]
@xMilotas it is expected that you still see the old paths. What we need to see is if it also contains a * and a remapped path behind it. GitLink simply remaps local files to remote urls. Basically something like this should be visible:

c:\source\mybuild\src\solutioninfo.cs*src/solutioninfo.cs
